src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -197,8 +197,8 @@ public int getDefaultCloudIndexPrefetchBufferSize() {
     // Intervals from command line (singleton for now)
     private List<ChromosomeInterval> intervals;
 
-    // Linked hash map between sample names and corresponding GVCF file name
-    private LinkedHashMap<String, Path> sampleNameToVcfPath = new LinkedHashMap<>();
+    // Sorted mapping between sample names and corresponding GVCF file name
```

Comment:
Add a prominent comment mentioning that maintaining the sample names in sorted order here is critical, and explain why!